packages/@webex/internal-plugin-metrics/src/metrics.types.ts

```diff
@@ -86,3 +95,55 @@ export type MediaQualityEventVideoSetupDelayPayload =
 export type SubmitMQEPayload = RecursivePartial<MediaQualityEvent['payload']> & {
   intervals: MediaQualityEvent['payload']['intervals'];
 };
+
+export type SubmitInternalEvent = ({
+  name,
+  payload,
+  options,
+}: {
+  name: InternalEvent['name'];
+  payload?: RecursivePartial<InternalEvent['payload']>;
+  options: any;
+}) => void;
```

Comment:
since you are defining the type only you do not need to deconstruct the parameter, so it would be just: 
```suggestion
export type SubmitInternalEvent = (params: {
  name: InternalEvent['name'];
  payload?: RecursivePartial<InternalEvent['payload']>;
  options: any;
}) => void;
```

same with the other definitions.

Also, can you define the `options` type, like you did it for `SubmitClientEvent`

Reply:
cheeky copy paste, refactored